pkg/k8s/report/table.go

```diff
@@ -42,10 +46,14 @@ func InfraColumns() []string {
 func (tw TableWriter) Write(report Report) error {
 	switch tw.Report {
 	case AllReport:
-		t := pkgReport.Writer{Output: tw.Output, Severities: tw.Severities, ShowMessageOnce: &sync.Once{}}
+		t := pkgReport.Writer{
+			Output:          tw.Output,
+			Severities:      tw.Severities,
+			ShowMessageOnce: &sync.Once{},
+		}
 		for _, r := range report.Resources {
 			if r.Report.Results.Failed() {
-				err := t.Write(r.Report)
+				err := t.Write(nil, r.Report)
```

Comment:
nit: maybe it better to remove context from `t.Write(nil, r.Report)` signature
or change the `func (tw TableWriter) Write(report Report)` to accept context to avoid passing `nil` value in `t.Write(nil, r.Report)`

Reply:
Nice catch. We must take context as an argument to implement an interface. I passed context.